src/core/shared_memory_manager.h

```diff
@@ -32,8 +32,21 @@
 #include "src/core/server_status.pb.h"
 #include "src/core/status.h"
 
+#ifdef TRTIS_ENABLE_GPU
+#include <cuda_runtime_api.h>
+#endif  // TRTIS_ENABLE_GPU
+
 namespace nvidia { namespace inferenceserver {
 
+#ifdef TRTIS_ENABLE_GPU
+typedef struct ipcCUDA_st {
```

Comment:
Why use a C-style struct with different naming convention? I'm guessing you found this somewhere but you need to adapt it to our coding standards. Should be within SharedMemoryManager class?

Reply:
Done